pkg/api/mlflow/dao/repositories/param.go

```diff
@@ -31,8 +32,50 @@ func NewParamRepository(db *gorm.DB) *ParamRepository {
 
 // CreateBatch creates []models.Param entities in batch.
 func (r ParamRepository) CreateBatch(ctx context.Context, batchSize int, params []models.Param) error {
-	if err := r.db.CreateInBatches(params, batchSize).Error; err != nil {
+	var err error
+	err = r.db.CreateInBatches(params, batchSize).Error
+	if err != nil {
+		// remove duplicate rows and try again
+		params, err1 := r.removeExactMatches(ctx, params)
+		if err1 != nil {
+			return eris.Wrap(err1, "error removing exact matches after: " + err.Error())
```

Comment:
please do not use `err.Error()`. We have to bubble up all the errors and stack trace.

Reply:
will wrap

Reply:
well at this point we have 2 errors to report and Wrap() does not seem to allow multiple error args

Reply:
now discards first error